### PR TITLE
Added call button

### PR DESCRIPTION
**Pull request by goke-epapa**
Since this is mobile first, I decided to add a call button so users can click the button to perform the transaction using the generated USSD code.

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -133,6 +133,8 @@
             <input id="code" class="code" readonly></input>
             <span id="render-copy">Click code to copy</span>
             <span id="render-copied" class="orange" style="display: none">Copied to clipboard!</span>
+            <span>Or</span>
+            <a id="call-link" href="#" class="button">Call</a>
```
Comment (azemoh) [inline index.html]: For some reason clicking this sends the generated code to the dialer without the hash (#)

Comment (fathermerry) [inline index.html]: Found this http://stackoverflow.com/questions/25890866/hash-key-stripped-from-ussd-code-in-tel-links-on-html-pages

**Comment by azemoh** [comment]
Thanks, this works for Android.

**Comment by fathermerry** [comment]
@azemoh are you making a pull request>

**Comment by azemoh** [comment]
Oh sure.

**Pull request merged**